test-alerts.test.cpp

```diff
@@ -0,0 +1,25 @@
+#define CATCH_CONFIG_MAIN  // This tells Catch to provide a main() - only do this in one cpp file
+
+#include "test/catch.hpp"
+#include "typewise-alert.h"
+#include "checker.h"
+
+TEST_CASE("Test checkAndAlert functionality")
+{
+    BatteryCharacter batteryChar;
+
+    batteryChar.coolingType = PASSIVE_COOLING;
+    REQUIRE(checkAndAlert(TO_CONTROLLER, batteryChar,-1) == 1);
```

Comment:
how do we verify if the intended message has been received by the right target, in this case controller.
Test should be present for that too